App/Screens/Account.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { View } from "react-native";
+import { withNavigation } from 'react-navigation';
+import { Card, Button, Text } from "react-native-elements";
+import { onSignOut } from "../Config/auth";
+
+
+export class Account extends Component {
+
+
+  //export default ({ navigation }) => (
+  render(){
+
+    const { navigation } = this.props;
+
+    return(
+      <View style={{ paddingVertical: 20 }}>
+        <Card title="John Doe">
+          <View
+            style={{
```

Comment:
yeah this was a complete grab from the tutorial I was following. I want to build this out to make it prettier and match the styling I have set. Once I actually I figure out authentication I'll be able to make it more personal with the person's initials.